### ISSUE TITLE
Code snippet makes plugin crash the eslint process

### ISSUE THREAD

**Issue by NickHeiner** [jest-community/eslint-plugin-jest#212]
The following snippet causes eslint-plugin-jest@22.1.0 to crash:

```js
        expect(querystring.parse(hash)).toBe.deep.equal({
            itemContext: 'trackId%3D99',
            pivotVideoId: 12
        });
```

The error is:

```
TypeError: Cannot read property '0' of undefined
    at argument (/Users/nheiner/code/tvui/node_modules/eslint-plugin-jest/rules/util.js:76:54)
    at expectToBeCase (/Users/nheiner/code/tvui/node_modules/eslint-plugin-jest/rules/util.js:34:3)
    at CallExpression (/Users/nheiner/code/tvui/node_modules/eslint-plugin-jest/rules/prefer-to-be-null.js:25:20)
    at listeners.(anonymous function).forEach.listener (/Users/nheiner/code/tvui/node_modules/eslint/lib/util/safe-emitter.js:45:58)
    at Array.forEach (<anonymous>)
    at Object.emit (/Users/nheiner/code/tvui/node_modules/eslint/lib/util/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/Users/nheiner/code/tvui/node_modules/eslint/lib/util/node-event-generator.js:251:26)
    at NodeEventGenerator.applySelectors (/Users/nheiner/code/tvui/node_modules/eslint/lib/util/node-event-generator.js:280:22)
    at NodeEventGenerator.enterNode (/Users/nheiner/code/tvui/node_modules/eslint/lib/util/node-event-generator.js:294:14)
    at CodePathAnalyzer.enterNode (/Users/nheiner/code/tvui/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:632:23)
```

When I remove that code snippet, the lint task completes successfully.

I know this isn't the latest version of eslint-plugin-jest, but I don't have time to test on the latest.

**Comment by macklinu** [comment]
I cannot reproduce this with a test case on master. It looks like #58 will resolve your issue, which was shipped in v22.1.1. Upgrading to the latest version of eslint-plugin-jest should get rid of this error for you.

Closing, but let us know if updating to the latest version of eslint-plugin-jest does not resolve for you. 👍

**Comment by NickHeiner** [comment]
I've confirmed that this is fixed in `v22.1.2`. Thanks!
